### DISCUSSION TITLE
Question about feature requested 2 years ago

### DISCUSSION THREAD

**Discussion by thrichardson-twist** [PyO3/maturin#2251] [Q&A]
Hi!

I'm mainly curious if for this: https://github.com/PyO3/maturin/discussions/1087

If you were ever able to get it to work for 'develop', as James mentioned we needed that for it to work in our use case.  Since then we've just never used Maturin for our code that requires the catboost model.

Since then we upgraded Maturin to quay.io/pypa/manylinux_2_28_x86_64 maybe 6 months ago and when I tested an attempt to get catboost working with it, it still fails.

Hoping maybe it did get implemented in a Maturin version somewhere and I just didn't notice.

**Comment by messense** [comment]
It should have been fixed in #1092, please open a new issue with a detailed repro steps.
